azkaban-common/src/main/java/azkaban/HadoopModule.java

```diff
@@ -47,79 +46,72 @@
  * needed.
  */
 public class HadoopModule extends AbstractModule {
-  private static final String CHTTP_SCHEME = "chttp";
-  private static final String LOCAL_SCHEME = "file";
-  private static final String HDFS_SCHEME = "hdfs";
 
-  private static final String HDFS_FS_IMPL_CONFIG = "fs.hdfs.impl";
```

Comment:
Do these two become externally configured instead of hardcoded -- is this the idea? Maybe a quick comment somewhere describing the essence of the change would help. Otherwise there are a lot of formatting changes, and it's not easy to see what's actually going on. Thanks!

Reply:
Sorry, I forgot to add a description after pushing the code changes.